### PR TITLE
markdown typo fixes

### PR DESCRIPTION
**Pull request by extemporalgenome**

### CODE REVIEW

**Review by ayazhafiz — DISMISSED** [review]

**Comment by extemporalgenome** [comment]
@ayazhafiz thanks for the review! I've fixed the conflict (which was only in a file which was removed from trunk/main since creating this PR)

File: design/language/RocStr.md
Comment (lukewilliamboswell) [inline design/language/RocStr.md:1]: I moved the str/README.md file here as it looks like more of a design doc, however I am not sure if this is the best place for it.

File: crates/compiler/README.md
Comment (lukewilliamboswell) [inline crates/compiler/README.md:1]: Was this file meant to be deleted?

**Comment by lukewilliamboswell** [comment]
I think there is a lot of good updates here. I have merged in main so hopefully it passes CI, but I am not the best person to review the Str design updates. @ayazhafiz I think you previously approved this PR, perhaps you could have another look at it?

File: crates/compiler/README.md
Comment (ayazhafiz) [reply crates/compiler/README.md:1]: Yes, can we add this file back?

**Review by ayazhafiz — COMMENTED** [review]
Thanks both; apologies for the delay, the sift through GH notifications is tough. Let's add the compiler README back and then LGTM.

**Comment by extemporalgenome** [comment]
@ayazhafiz @lukewilliamboswell I've added back the file `crates/compiler/README.md`. I recall removing some file which I had previously edited, after a merge from main to resolve conflicts, because it looked like it had been removed from main since my initial edits. I don't recall if this was the file or not.

**Review by ayazhafiz — APPROVED** [review]

**Comment by extemporalgenome** [comment]
Oops, it looks like I didn't sign my commits. @lukewilliamboswell and @ayazhafiz how do you feel about me force-pushing a rebased, fully signed set of commits over this one, or creating a new PR on a new branch with verifiably the same overall patch content (aside from signing)? If so, @lukewilliamboswell would you want to push your changes back atop mine, or would I just try to sign your commits as part of my rebase, maintaining your authorship?

**Comment by ayazhafiz** [comment]
Force-pushing a new set is fine with me, but since Luke's commits are on here I'll let him make the call

**Comment by lukewilliamboswell** [comment]
I don't mind either way. If you can make it all happen in one push then I'd suggest we do that 😄

**Comment by extemporalgenome** [comment]
@ayazhafiz I've created https://github.com/roc-lang/roc/pull/5272 to supersede this one. You can check equivalence after ensuring main is merged into at least this original PR (there should be no differences at that point, thus the new PR should contain the changes that you already reviewed).

**Comment by rtfeldman** [comment]
Closing in favor of https://github.com/roc-lang/roc/pull/5272

**Pull request closed without merging**